### PR TITLE
Replace deprecated font.getsize() to make it compatible with pillow 10.2.0

### PR DESCRIPTION
**Pull request by SamuelWiet**
The deprecated method was removed with pillow 10 but could replaced by the implemented method.

### CODE REVIEW

File: fentoboardimage/Coordinates.py [hunk 1]
```diff
@@ -4,33 +4,35 @@
 from .Utils import flippedCheck, indicesToSquare
 
 
+def get_size(font, s):
+    left, top, right, bottom = font.getbbox(s)
+    width, height = right - left, bottom
```
Comment (ReedKrawiec) [inline fentoboardimage/Coordinates.py:9]: nit: Is top guaranteed to be 0? I'm wondering if height should be `bottom - top`? Looks like the font's origin [sometimes can be non-zero.](https://stackoverflow.com/a/76234689)

**Review by ReedKrawiec — CHANGES_REQUESTED** [review]
These changes make sense to me. Tests are passing, so once that small edge case is addressed I'm happy to merge. I appreciate the contribution!

EDIT: merging after learning more context on the underlying deprecation

File: fentoboardimage/Coordinates.py (hunk 1 above)
Comment (ReedKrawiec) [reply fentoboardimage/Coordinates.py:9]: Update: looks like the old method had an issue that ignored the top property, so this is accurate to the old implementation.
